default/stringtables/en.txt

```diff
@@ -9066,7 +9069,7 @@ Colonization
 PC_COLONY_DESC
 '''Colony parts integrate with a ship to provide the necessary equipment to establish ownership on a new planet.
 
-These parts may also increase the [[metertype METER_POPULATION]] on a ship.
+These parts may also increase the [[metertype METER_POPULATION]] on a ship (referred to as Colonists).
```

Comment:
are they referred to as colonists much outside of these edits?

Reply:
Mainly in the Fleet Window panel: 
![colonists](https://user-images.githubusercontent.com/11946570/52179364-ee3ef280-27d9-11e9-83ed-36d522f685f1.gif)

It's to make clear for new players, that Colonists are the Population value for a ship, and works in the same way.